docs/management/connectors/action-types/servicenow-sir.asciidoc

```diff
@@ -7,6 +7,64 @@
 
 The ServiceNow SecOps connector uses the https://docs.servicenow.com/bundle/orlando-application-development/page/integrate/inbound-rest/concept/c_TableAPI.html[V2 Table API] to create ServiceNow security incidents.
 
+[float]
+[[servicenow-sir-connector-prerequisites]]
+==== Prerequisites
+After upgrading to 7.16, you must complete the following within your ServiceNow instance before creating a new ServiceNow SecOps connector or <<servicenow-sir-connector-update, updating an existing one>>:
+
+* Install Elastic for Security Operations (SecOps) from the https://store.servicenow.com/sn_appstore_store.do#!/store/home[ServiceNow Store].
```

Comment:
With a very high probability, this will be the link for SecOps in the store: https://store.servicenow.com/sn_appstore_store.do#!/store/application/2f0746801baeb01019ae54e4604bcb0f